bindings/matlab/+iDynTreeWrappers/generalizedBiasForces.m

```diff
@@ -0,0 +1,36 @@
+function h = generalizedBiasForces(KinDynModel)
+
+    % GENERALIZEDBIASFORCES retrieves the generalized bias forces from 
+    %                            the reduced model. 
+    %
+    % This matlab function wraps a functionality of the iDyntree library.                     
+    % For further info see also: http://wiki.icub.org/codyco/dox/html/idyntree/html/
+    %
+    % FORMAT:  h = generalizedBiasForces(KinDynModel)
+    %
+    % INPUTS:  - KinDynModel: a structure containing the loaded model and additional info.
+    %
+    % OUTPUTS: - h: [ndof+6 x 1] generalized bias accelerations.
+    %
+    % Author : Gabriele Nava (gabriele.nava@iit.it)
+    % Genova, Nov 2018
```

Comment:
Is this location/date info necessary? It seems something that is tricky to keep updated. If possible, I would prefer to remove it.